corehq/apps/app_manager/app_translations/app_translations.py

```diff
@@ -944,9 +942,8 @@ def _update_id_mappings(rows, detail):
                              .format(detail.field)))
             else:
                 for row in rows:
-                    prop = row['case_property'].rstrip(ID_MAPPING_SUFFIX)
-                    if prop in mappings_by_prop:
-                        _update_translation(row, mappings_by_prop[prop].value)
```

Comment:
This is a simplification since I realized that `row['id']` already has what I needed